tests/components/songpal/test_media_player.py

```diff
@@ -12,6 +12,7 @@
     SongpalException,
     VolumeChange,
 )
+from songpal.notification import SettingChange
```

Comment:
This could also be done inside the `from songpal import` block. Other than that, I have no more suggestions at the moment.

Reply:
Not sure to understand how?
Because currently SettingChange is not exported, no the only way I see is like
```python
from songpal import (
    ConnectChange,
    ContentChange,
    PowerChange,
    SongpalException,
    VolumeChange,
    notification
)
```
And in the code use `notification.SettingChange`, but personally I find the "double from" better.
Did I miss an other way to do it?

Reply:
I meant directly importing `SettingChange` from `songpal` package, as it's part of the public API. It's a very minor nit, and there's necessary no need for action, especially as this is (just) test code.